soc/arm/nuvoton_npcx/common/pinctrl_soc.h

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright (c) 2022 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _NUVOTON_PINCTRL_SOC_H_
+#define _NUVOTON_PINCTRL_SOC_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
```

Comment:
nit: is this file included by any C++ code?

Reply:
Done.